ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -278,6 +278,7 @@
       "CONFIG_whisk_invoker_https_keystorePassword": "{{ invoker.ssl.keystore.password }}"
       "CONFIG_whisk_invoker_https_keystoreFlavor": "{{ invoker.ssl.storeFlavor }}"
       "CONFIG_whisk_invoker_https_clientAuth": "{{ invoker.ssl.clientAuth }}"
+      "CONFIG_whisk_containerPool_prewarmExpiredCheckPeriod": "{{ container_pool_prewarmExpiredCheckPeriod | default('1 minute') }}"
```

Comment:
It's a bit picky but how about changing the name to a more intuitive one such as `container_pool_prewarm_expirationCheckInterval`?

Reply:
Great, updated accordingly.